python/gtsam/utils/plot.py

```diff
@@ -113,7 +117,16 @@ def plot_point2_on_axes(axes,
                         linespec: str,
                         P: Optional[np.ndarray] = None) -> None:
     """
-    Plot a 2D point on given axis `axes` with given `linespec`.
+    Plot a 2D point and its corresponding uncertainty ellipse on given axis
+    `axes` with given `linespec`.
+
+    Based on Stochastic Models, Estimation, and Control Vol 1 by Maybeck,
+    page 366
+    For the 2D case:
+    k = 1.515 corresponds to 1 std, 68.26% of all probability
```

Comment:
I would an argument k here and say
k = 1 corresponds to xx%
k = 2 corresponds to xx%
k = 2.5 (default) corresponds to xx%